package.json

```diff
@@ -169,12 +169,13 @@
     "@chainsafe/bls": "5.1.1",
     "@chainsafe/bls-keygen": "0.3.0",
     "@chainsafe/bls-keystore": "2.0.0",
-    "@chainsafe/lodestar-beacon-state-transition": "0.13.0",
-    "@chainsafe/lodestar-config": "0.13.0",
-    "@chainsafe/lodestar-db": "0.13.0",
-    "@chainsafe/lodestar-types": "0.13.0",
-    "@chainsafe/lodestar-utils": "0.13.0",
-    "@chainsafe/lodestar-validator": "0.13.0",
+    "@chainsafe/lodestar-beacon-state-transition": "0.16.0",
+    "@chainsafe/lodestar-config": "0.16.0",
+    "@chainsafe/lodestar-db": "0.16.0",
+    "@chainsafe/lodestar-types": "0.16.0",
+    "@chainsafe/lodestar-utils": "0.16.0",
+    "@chainsafe/lodestar-validator": "0.16.0",
+    "@chainsafe/persistent-ts": "0.16.0",
```

Comment:
do we use that directly somewhere or some package is missing dependency delcaration?

Reply:
one of the packages use `@chainsafe/persistent-ts`

Reply:
but it will be included automatically no?

Reply:
probably package that uses it doesn't have it as a dependency

Reply:
do you have error so I can figure out where is it missing, so we can open issue. Don't push issues under the rug 😄

Reply:
done https://github.com/ChainSafe/lodestar/issues/2052